app/src/test/java/org/simple/clinic/instantsearch/InstantSearchResultsItemTypeTest.kt

```diff
@@ -0,0 +1,184 @@
+package org.simple.clinic.instantsearch
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.TestData
+import org.simple.clinic.patient.PatientSearchResult
+import org.simple.clinic.widgets.PatientSearchResultItemView
+import java.util.UUID
+
+class InstantSearchResultsItemTypeTest {
+
+  private val currentFacilityUuid = UUID.fromString("69cf85c8-6788-4071-b985-0536ae606b70")
+  private val otherFacilityUuid = UUID.fromString("28ed7dca-828e-4178-942f-fd6937f37155")
+  private val currentFacility = TestData.facility(currentFacilityUuid)
+
+  @Test
+  fun `list items must be generated from the search results`() {
+    // given
+    val patientUuid1 = UUID.fromString("1d5f18d9-43f7-4e7f-92d3-a4f641709470")
```

Comment:
Can we extract out common variables like `patientUuid1`, `patientUuid2` to class-level variables, please?

Reply:
Not needed in this scenario. Because we are constructing the patient search results in individual tests, even though a couple of tests have the same UUIDs they are meant to be different patient search results. So it's fine here.